[2024-03-20 22:51:07,324] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.compute_rf_score 2024-03-20T22:50:00+00:00 [queued]>
[2024-03-20 22:51:07,339] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.compute_rf_score 2024-03-20T22:50:00+00:00 [queued]>
[2024-03-20 22:51:07,339] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-03-20 22:51:07,339] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2024-03-20 22:51:07,339] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-03-20 22:51:07,356] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): compute_rf_score> on 2024-03-20T22:50:00+00:00
[2024-03-20 22:51:07,366] {standard_task_runner.py:52} INFO - Started process 29482 to run task
[2024-03-20 22:51:07,370] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', '***_exam', 'compute_rf_score', '2024-03-20T22:50:00+00:00', '--job-id', '7802', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmp070va892', '--error-file', '/tmp/tmpmjfbzva_']
[2024-03-20 22:51:07,371] {standard_task_runner.py:77} INFO - Job 7802: Subtask compute_rf_score
[2024-03-20 22:51:07,425] {logging_mixin.py:104} INFO - Running <TaskInstance: ***_exam.compute_rf_score 2024-03-20T22:50:00+00:00 [running]> on host 3a3d0401ac26
[2024-03-20 22:51:07,499] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=***_exam
AIRFLOW_CTX_TASK_ID=compute_rf_score
AIRFLOW_CTX_EXECUTION_DATE=2024-03-20T22:50:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-20T22:50:00+00:00
[2024-03-20 22:51:10,285] {python.py:151} INFO - Done. Returned value was: None
[2024-03-20 22:51:10,301] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=***_exam, task_id=compute_rf_score, execution_date=20240320T225000, start_date=20240320T225107, end_date=20240320T225110
[2024-03-20 22:51:10,352] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-03-20 22:51:10,400] {local_task_job.py:151} INFO - Task exited with return code 0
